app/src/components/ScheduleBuilder.tsx

```diff
@@ -227,8 +224,10 @@ export default class ScheduleBuilder extends Component<Props, States> {
                         }
                         disabled={false}
                     >
-                        {LIMITS.map(([num, str]) => (
-                            <option value={num}>Generate ≤{str}</option>
+                        {LIMITS.map(([num, str], idx) => (
+                            <option value={num} key={idx}>
```

Comment:
Same reason as above.

Good work finding these missing keys via the console.

Reply:
I think it's OK here because LIMITS is a constant so we don't have to worry about the content of the array being changed.

Reply:
Immutable so yup.

Nit: Use the value (`num`) for the key.